source/OutfitterPanel.cpp

```diff
@@ -261,18 +261,33 @@ int OutfitterPanel::DetailWidth() const
 
 int OutfitterPanel::DrawDetails(const Point &center)
 {
-	string selectedItem = "Nothing Selected";
+	const string selectedItem = selectedOutfit ? selectedOutfit->DisplayName() : "Nothing Selected";
 	const Font &font = FontSet::Get(14);
 	const Color &bright = *GameData::Colors().Get("bright");
 	const Color &dim = *GameData::Colors().Get("medium");
 	const Sprite *collapsedArrow = SpriteSet::Get("ui/collapsed");
 
-	int heightOffset = 20;
+	int heightOffset = 0;
+
+	if(planet && !planet->OutfitterDescription().empty())
+	{
+		Point descriptionPoint(center.X() - INFOBAR_WIDTH / 2 + 25, center.Y() + heightOffset);
```

Comment:
heightoffset is always 0 here?

Reply:
I added the offset to each initial point and then increment the offset after draw. That way the draw calls can be moved around without issue.